recipes/newrelic/infrastructure/suse.yml

```diff
@@ -54,6 +54,15 @@ install:
             echo "awk is required to run the newrelic install. Please install awk and re-run the installation." >> /dev/stderr
             exit 12
           fi
+        - |
+          IS_INFRA_AVAILABLE=$(curl -Is https://download.newrelic.com/infrastructure_agent/linux/zypp/sles/{{.SLES_VERSION}}/x86_64/newrelic-infra.repo | grep " 200 " | wc -l)
+          if [ $IS_INFRA_AVAILABLE -eq 0 ] ; then
+            echo "there is no newrelic infrastructure agent available for the distribution '{{.SLES_VERSION}}'." >> /dev/stderr
+            exit 13
+          fi
+      vars:
+        SLES_VERSION:
+          sh: cat /etc/os-release | grep VERSION_ID | awk -F = '{print $2}'
```

Comment:
Not a blocker, but `awk` can do all of this: `awk -F= '/VERSION_ID/ {print $2}' /etc/os-release`

Reply:
Great tip, I'll update. Thank you